src/uniprotkb/components/protein-data-views/XRefView.tsx

```diff
@@ -106,10 +106,11 @@ const EMBLXref: FC<{
   const genBankInfo = databaseToDatabaseInfo.GenBank;
   const ddbjInfo = databaseToDatabaseInfo.DDBJ;
   const { properties, additionalIds } = xref;
-  if (!genBankInfo || !ddbjInfo) {
-    logging.warn(
-      'GenBank or DDBJ database information not found in drlineconiguration'
+  if (!databaseInfo?.uriLink || !genBankInfo?.uriLink || !ddbjInfo?.uriLink) {
+    logging.error(
+      'EMBL, GenBank or DDBJ database information not found in drlineconiguration'
```

Comment:
Is it meant to say `drlineconfiguration` (with "f")?

Reply:
Good catch! Also, drlineconfiguration has been renamed to allDatabases ~1year ago so will change this to the following to guard against future file name changes:
```
EMBL, GenBank or DDBJ database information not found in database configuration file
```